src/components/Listhome.js

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { getIngredients, pickIngredient } from '../redux/home/home';
+
+function Listhome() {
+  const dispatch = useDispatch();
+  const fetchApi = 'https://www.thecocktaildb.com/api/json/v1/1/list.php?i=list';
```

Comment:
- [ ] Your app is working great 👏 But it seems that you forgot one main feature which is the stateless filtering component that it's supposed to filter the fetched data from the API using some parameters like the category name for example. 😄 🙏 Kindly let's work on that!
_If you need a reminder please see the project requirements [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=When%20the%20page%20loads%2C%20the%20webapp%20shows%20the%20list%20of%20categories%20that%20could%20be%20filtered%20out%20by%20some%20parameter%2C%20for%20example%20by%20the%20category%20name.)._

Reply:
made filter for the ingredients with a form